### ISSUE TITLE
Query log: low contrast in small text

### ISSUE THREAD

**Issue by reynir** [robur-coop/dnsvizor#83]
> The contrast between the small text and background is not sufficient. The contrast ratio is 2.94:1 when it should be 4.5:1 for text smaller than 18px.

<img width="172" height="29" alt="Image" src="https://github.com/user-attachments/assets/cdbe494d-da89-494b-aa5d-67072c0eed68" />

[1.4.3 Contrast (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum)
